.pre-commit-config.yaml

```
@@ -22,3 +22,7 @@ repos:
     hooks:
     - id: mypy
       additional_dependencies: [types-setuptools, types-PyYAML]
+      # This intentionally counters the settings in mypy.ini to allow a loose local
+      # check and a strict CI check.  This difference may or may not be retained long
```

```suggestion
      # check and a strict CI check. This difference may or may not be retained long
```

nit, so you may ignore it :)